src/figuresClasses.ts

```diff
@@ -1,11 +1,68 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
+  color: string;
```

Comment:
You need to describe all cases for shape and color, for example `'red' | 'green' | 'blue'`